vis-registers.c

```diff
@@ -188,6 +199,7 @@ bool register_resize(Register *reg, size_t count) {
 enum VisRegister vis_register_from(Vis *vis, char reg) {
 	switch (reg) {
 	case '+': return VIS_REG_CLIPBOARD;
+	case '*': return VIS_REG_PRIMARY;
```

Comment:
Instead of adding this you can remove the entry above for `+`. Both cases should be covered by the for loop further down.